packages/xstate-wallet/src/config.ts

```diff
@@ -30,6 +30,8 @@ export const ETH_ASSET_HOLDER_ADDRESS: string = process.env.ETH_ASSET_HOLDER_ADD
 export const HUB_ADDRESS: string =
   process.env.HUB_ADDRESS || '0xaaaa84838319627Fa056fC3FC29ab94d479B8502';
 
+// This account corresponds to 0x187bb12e927c1652377405f81d93ce948a593f7d66cfba383ee761858b05921a
+// That key is provided eth in @statechannels/devtools/utils/startGanache.js
 export const HUB_DESTINATION = (process.env.HUB_DESTINATION ||
   hexZeroPad('0x8199de05654e9afa5c081bce38f140082c9a7733', 32)) as Destination;
```

Comment:
Could this address instead be this?
- the address corresponding to `HUB_CHAIN_PK`, if that environment variable is defined

Reply:
Not a bad idea. What if `HUB_CHAIN_PK` is not defined?

Reply:
Then we could default to `HUB_ADDRESS` (like now).

Or, we could throw?

Reply:
10f36f589

Reply:
I ended up reverting the implementation of this suggestion. `HUB_DESTINATION` constant/env var has to be defined for `web3torrent`. Of course, `web3torrent` can calculate the destination from the private key. But this seems wrong since `web3torrent` is unaware of private keys at the moment.

If `HUB_DESTINATION` is defined for `web3torrent`, might as well use the env var for the wallet as well.